packages/flet/lib/src/models/multi_view.dart

```diff
@@ -0,0 +1,16 @@
+import 'dart:ui';
+
+class MultiView {
```

Comment:
From what I see, `MultiView` is web-only. To reduce ambiguity, will it make sense to have "Web" in it's name (everywhere)?

Reply:
Well, "WebView" sounds reasonable, but we already have "WebView", right? :) Could be confusing to folks.